about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
```

Comment:
- [ ] After the Header section and before the section with two images, there should be a section that contains: a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo. Kindly make sure to implement this in your website so it includes all the sections from the original design

|Your website|Provided design|
|-|-|
|![Screenshot_20230706_202046](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/a7a13b69-faa0-439b-9bb2-20fc3c49212c)|![Screenshot_20230706_202107](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/8414fcaa-af95-4d57-b8c5-1252494ad021)|